usecases/connect-autonomous-database/src/main/java/com/oracle/cloud/world/demo/model/Country.java

```diff
@@ -0,0 +1,42 @@
+package com.oracle.cloud.world.demo.model;
```

Comment:
Suggested to remove Cloud World reference (generalize)...

Reply:
Renamed